nocopy_linkbuffer.go

```diff
@@ -285,9 +293,9 @@ func (b *LinkBuffer) Slice(n int) (r Reader, err error) {
 	b.recalLen(-n) // re-cal length
 
 	// just use for range
-	p := &LinkBuffer{
-		length: int64(n),
-	}
+	p := new(LinkBuffer)
```

Comment:
May I ask, what are the differences between these two approaches~ thanks
i think they are same?
```
p := &LinkBuffer{
		length: int64(n),
	}
```
```
p := new(LinkBuffer)
p.length = int64(n)
```

Reply:
@kingcanfish no different just to make code more clean